recipes/sophios/conda_build_config.yaml

```diff
@@ -0,0 +1,5 @@
+channel_sources:
+ - conda-forge,defaults,bioconda
```

Comment:
You cannot rely on other channels to build on conda-forge.

Reply:
we have a dependency on `toil` that is not available in conda-forge. How should we tackle that?

Reply:
Add tail to conda-forge.